vip-helpers/class-vip-media-library-caching.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * Media library caching helpers.
+ * 
+ * @package vip-helpers
+ */
+
+ // phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+
+namespace Automattic\VIP\Helpers;
+
+/**
+ * VIP Media Library Caching class.
+ */
+class VIP_Media_Library_Caching {
```

Comment:
If we use object cache, it would be good to use a non-default cache group as well so we can better track stats, and future-proofing if we ever support `wp_cache_flush_group()`

Reply:
This was implemented, and a `mime_types` cache group is being used across this class.